test/integration/policyConsumptionTest.ts

```diff
@@ -51,42 +52,37 @@ describe("Test: that policy lists are consumed by the associated synapse module"
         if(this.config.web.ruleServer.enabled) {
             this.timeout(5000)
             LogService.debug('policyConsumptionTest', `Rules at end of test ${JSON.stringify(await currentRules(this.mjolnir), null, 2)}`);
-            const mjolnir = config.RUNTIME.client!;
             // Clear any state associated with the account.
-            await mjolnir.setAccountData('org.matrix.mjolnir.watched_lists', {
+            await this.mjolnir.client.setAccountData('org.matrix.mjolnir.watched_lists', {
                 references: [],
             });
         }
     })
     this.beforeAll(async function() {
-        if (!this.config.web.ruleServer.enabled) {
+        let config = configRead();
+        if (!config?.web?.ruleServer?.enabled) {
```

Comment:
I believe that `this.mjolnir` will exist in the beforeAll hook, but this works too

Reply:
In my experience, it didn't seem to exist.